### PR TITLE
Reduce workload and test payload from release-controller

### PR DESCRIPTION
**Pull request by deepsm007**
Only Informing jobs
/cc @Prashanth684 
/cc @bradmwilliams

### CODE REVIEW

**Comment by deepsm007** [comment]
Test jobs are expected to fail due to resource crunch

File: core-services/release-controller/_releases/release-ocp-4.10-arm64.json [hunk 1]
```diff
@@ -13,9 +13,29 @@
     "tag":{"tagRef":{"name":"4.10"}}
   },
   "verify":{
-    "arm64":{
+    "aws-arm64":{
```
Comment (yselkowitz) [inline core-services/release-controller/_releases/release-ocp-4.10-arm64.json:16]: Do these names have to have arm64 in them?

File: core-services/release-controller/_releases/release-ocp-4.10-arm64.json [hunk 2]
```diff
@@ -13,9 +13,29 @@
...
       "optional":true,
       "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.10-ocp-e2e-aws-arm64"}
+    },
+    "serial-aws-arm64":{
```
Comment (yselkowitz) [inline core-services/release-controller/_releases/release-ocp-4.10-arm64.json:20]: Assuming arm64 has to remain, this should be either aws-serial-arm64 or aws-arm64-serial, and make the rest of these names consistent.

File: core-services/release-controller/_releases/release-ocp-4.10-arm64.json [hunk 3]
```diff
@@ -13,9 +13,29 @@
...
+      "optional":true,
+      "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.10-ocp-e2e-serial-aws-arm64"}
+    },
+    "aws-ovn-arm64":{
```
Comment (yselkowitz) [inline core-services/release-controller/_releases/release-ocp-4.10-arm64.json:24]: Or (if arm64 has to remain) aws-arm64-ovn, it should be consistent.

File: core-services/release-controller/_releases/release-ocp-4.9-arm64.json [hunk 4]
```diff
@@ -13,13 +13,9 @@
     "tag":{"tagRef":{"name":"4.9"}}
   },
   "verify":{
-    "e2e-aws-arm64":{
+    "aws-arm64":{
```
Comment (yselkowitz) [inline core-services/release-controller/_releases/release-ocp-4.9-arm64.json:16]: Ditto here wrt "arm64".

File: core-services/release-controller/_releases/release-ocp-4.10-arm64.json (hunk 1 above)
Comment (deepsm007) [reply core-services/release-controller/_releases/release-ocp-4.10-arm64.json:16]: yes, according to Dennis we need to specify we are running arm64 on aws.

**Review by yselkowitz — APPROVED** [review]
/lgtm

**Pull request merged**